package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
I would agre  but file is fully read just after using isbinaryfile.

@see https://github.com/yeoman/generator/pull/737#issuecomment-70362446.